INSTALL

```diff
@@ -26,7 +27,16 @@ Prerequisites
 
 - libpng 1.2.x or later
 
-- wxPython 4.0.x or later
+  For example::
+      
+          sudo apt-get install -y 
```

Comment:
Does this really work with splitting it across multiple lines without escaping the newlines?